### PR TITLE
Update dates in footer of index.html

### PR DESCRIPTION
**Pull request by SaakiBhojanam**
I made some basic updates to the UI and grammar of index.html

### CODE REVIEW

**Comment by geosir** [comment]
Hi @SaakiBhojanam, thanks for your proposals! This is a good start - however, I would like to encourage you to go farther and expand your contribution, as PRs should represent a substantial improvement (for example, a bugfix or coherent redesign).

PRs should also focus on one specific theme rather than a collection of different changes. This helps to organize development as a timeline of distinct steps forward, making it very clear to future contributors what the PR has done.

Please submit a new PR when you're ready. If you have ideas for changes and want to have a longer discussion, you can open an Issue, which is better for open-ended discussion.

File: index.html [hunk 1]
```diff
@@ -298,8 +298,8 @@ <h2>Your Schedule</h2>
     </div>
     <p id="legal">
         George's Schedule Machine v4.0.<br/>
-        Copyright &copy; 2014-2018 <a href="https://george.moe">George Moe</a><br/>
-        Copyright &copy; 2022 <a href="contributors.html">George's Schedule Machine Contributors</a><br/>
```
Comment (geosir) [inline index.html:302]: Under copyright convention, these dates are correct. The work between 2014-2018 was mine. In 2022, there were contributions from others, which are represented by the Contributors. When a contribution is made in 2023, the year range would be updated to say 2022-2023.

File: index.html [hunk 2]
```diff
@@ -92,7 +92,7 @@ <h2>Schedule Commander</h2>
                 <!-- There is no official MDL selector, so need to find a working and nice element for this. -->
                 <div class="mdl-cell mdl-cell--12-col">
                     <label for="label-color">Label Color</label>
-                    <select id="label-color" class="u-full-width">
+                    <select id="label-color" class="u-full-width mdl-textfield__input" style="border-radius: 8px;">
```
Comment (geosir) [inline index.html:95]: For style changes, please include before/after screenshots and a description of the change.

This is a good change and I would welcome just this as PR of its own: Updating the selector to use the MDL style that is now available is a substantial fix over the current default selector. However, please leave out the border radius change, as none of the other elements have borders that round -- unless you have an idea for applying rounded borders as a consistent design overhaul to the rest of the application.

Comment (SaakiBhojanam) [reply index.html:95]: Ok, I will make that change. I'm quite new to GitHub and was just excited to help out the open-source community.

**Pull request closed without merging**